src/commands/integration-test-create-cluster.yaml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  kind-config:
+    type: string
+steps:
+  - unless:
+      condition: << parameters.kind-config >>
+      steps:
+        - run:
+            name: "Prepare default kind config"
```

Comment:
Let's please add the command prefix to the job names like here https://github.com/giantswarm/architect-orb/blob/29f86bd77f1ad5de0fca201f85148fb1d853af18/src/commands/go-test-legacy.yaml#L3
That makes it easier to realise what comes from where when all that is concatenated into a single yaml.